pkg/util/constants.go

```diff
@@ -232,6 +232,36 @@ const (
 	NamespaceKarmadaSystem = "karmada-system"
 )
 
+// Addon and component names used within the Karmada ecosystem.
+const (
+	// KarmadaDeschedulerAddonName is the name of the Karmada Descheduler addon.
+	KarmadaDeschedulerAddonName = "karmada-descheduler"
```

Comment:
Do we need to clarify these constants to components and addon?
Can we do it simply, like:
```suggestion
	// KarmadaDeschedulerComponentName is the name of the Karmada Descheduler.
	KarmadaDeschedulerComponentName = "karmada-descheduler"
```